caikit/runtime/servicers/training_management_servicer.py

```diff
@@ -131,7 +131,7 @@ def _get_model_future(training_id: str, operation: str):
         Wrapped here so that we only catch errors directly in the `trainer.get_model_future` call
         """
         try:
-            return MODEL_MANAGER.get_model_future(training_id=training_id)
+            return MODEL_MANAGER.get_model_future(training_id)
```

Comment:
Why remove this kwarg? I think this may be residual from the earlier code where they were merged, but probably better to revert this now that they're split.